holmes/core/supabase_dal.py

```
@@ -410,6 +411,69 @@ def get_issue_data(self, issue_id: Optional[str]) -> Optional[Dict]:
+
+        row = res.data[0]
+        id = row.get("runbook_id")
+        symptom = row.get("symptoms")
+        title = row.get("subject_name")
+        raw_instruction = row.get("runbook").get("instructions")
+        try:
+            if isinstance(raw_instruction, list) and len(raw_instruction) == 1:
+                instruction = raw_instruction[0]
+            else:
+                instruction = json.loads(raw_instruction)[0]
+        except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)